src/components/addTodoForm/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react"
+import { useDispatch } from "react-redux"
+import { addNewTodo, isAdding } from "redux/actions"
+import ImageButton from "components/buttons/imageButton"
+import "components/addTodoForm/form.scss"
+import { customSanitize } from "utils/sanitizeInput"
+import { DELETE_BUTTON_ALT, IMAGE_PATHS } from "utils/constants"
```

Comment:
Let's follow a order in terms of importing throughout our project:
Package imports comes first then local and lastly css imports.
And between Package and Local import lets follow this order: default imports comes first and then comes the named ones. Thanks.

```suggestion
import React, { useState } from "react"
import { useDispatch } from "react-redux"
import { addNewTodo, isAdding } from "redux/actions"
import ImageButton from "components/buttons/imageButton"
import { customSanitize } from "utils/sanitizeInput"
import { DELETE_BUTTON_ALT, IMAGE_PATHS } from "utils/constants"
import "components/addTodoForm/form.scss"
```